src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationAndGeneratorDriverTranslationAction.cs

```diff
@@ -26,7 +26,7 @@ public virtual Task<Compilation> TransformCompilationAsync(Compilation oldCompil
             /// (like <see cref="ReplaceAllSyntaxTreesAction"/>) will unilaterally remove all trees, and would have unexpected
             /// side effects. This opts those out of operating on ones with generated documents where there would be side effects.
             /// </remarks>
```

Comment:
```suggestion
            /// </remarks>
            /// <value>
            /// The value only affects cases where the application of the translation action does not affect the results
            /// of source generators active for a project. In the context of this condition, the property value should be
            /// <see langword="true"/> if the translation action is unordered with respect to the application of source
            /// generators; otherwise, <see langword="false"/> if the translation action must run before source
            /// generators to produce a correct final compilation.
            /// </value>
```

Reply:
I'm not sure I understand the proposal -- there's nothing to do with ordering here.